src/containers/login/index.tsx

```diff
@@ -0,0 +1,115 @@
+import 'font-awesome/css/font-awesome.min.css';
+
+import React, { useEffect, useState } from 'react';
+import { Spinner } from "react-bootstrap";
+import { useSelector, useDispatch } from '../../store/index'; 
+import {loginUser} from "../../store/actions/login";
+import { useNavigate } from "react-router-dom";
+import profile from "..//../component/image/undraw_login_re_4vu2.svg";
+
+import '..//../styles/sass/main.scss'
+
+
+
+export const LoginPage = () => {
+  const {error,userLoggedIn} = useSelector((state) => state.user)
+  const dispatch = useDispatch()
```

Comment:
```suggestion
  const dispatch = useDispatch();
```
There should be `;` at the end of each line. Add an `eslint-rule` for this.